src/config/ConfigManager.cpp

```diff
@@ -469,6 +469,80 @@ void CConfigManager::handleRawExec(const std::string& command, const std::string
     g_pKeybindManager->spawn(args);
 }
 
+static bool parseModeLine(const std::string& modeline, drmModeModeInfo& mode) {
+    std::vector<std::string> args;
+    std::stringstream ss;
+
+    for (auto it = modeline.begin();; it++) {
```

Comment:
useless, you can use `CVarList(modeline, 0, ' ')`

Reply:
I thought about it, but the problem with CVarList is that only one character can be a separator. Someone trying to use tabs might get an unexpected surprise (which is also inconsistent with the parser's behavior in other contexts, e.g. even CVarList itself strips both tabs and spaces *around* the separators).

If you are fine with that I can remove the complicated loop, though in my personal opinion it's poor UX.

Reply:
or you could've just added the functionality to cvarlist. Anyways, I've done that. Rebase against master and use `'s'` as the separator